code/game/gamemodes/cult/ritual.dm

```diff
@@ -12,6 +12,17 @@ This file contains the arcane tome files.
 	throw_speed = 2
 	throw_range = 5
 	w_class = WEIGHT_CLASS_SMALL
+	var/list/rune_types //Rune types that we can draw
```

Comment:
why not make it static+check for LAZYLEN(rune_types), so that it only makes a list of runes once?